css-ecommerce-page/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container container-background-color MT-10-percent">
+    <div class="row">
+      <div class="column-full">
+        <div class="tAC">
+          <h2>New Arrivals</h2>
+        </div>
+      </div>
+    </div>
+    <div class="row padding-LR">
+      <div class="column-one-fifth padding-LR">
+        <img src="images/dress1.jpeg" alt="Black Dress">
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <img src="images/ysl-bag.jpeg" alt="Yves Saint Laurent Bag">
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <img src="images/key.jpeg" alt="Tiffany Key">
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <img src="images/lv-bag.jpg" alt="Louis Vuitton">
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <img src="images/dress2.jpeg" alt="Blue and White Dress">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-fifth padding-LR">
+        <p>Free People <span class="gray-color">| Size 6</span></p>
+        <p><span class="money-background-color">$35</span> <s>$125 retail</s></p>
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <p>Yves Saint laurent <span class="gray-color">| One-Size</span></p>
+        <p> <span class="money-background-color">$200</span> <s>$1600 retail</s></p>
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <p>Tiffany <span class="gray-color">| One-Size</span></p>
+        <p><span class="money-background-color">$476</span> <s>$2000 retail</s></p>
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <p>Louis Vuitton <span class="gray-color">| One-Size</span></p>
+        <p><span class="money-background-color">$95</span> <s>$1200 retail</s></p>
+      </div>
+      <div class="column-one-fifth padding-LR">
+        <p>Reformation <span class="gray-color">| Size-4</span></p>
+        <p><span class="money-background-color">$20</span> <s>$198 retail</s></p>
+      </div>
```

Comment:
These images and the text describing them are related to one another, so they should not be in separate rows. Please make each column for the content hold the image as well as the text describing it.